woocommerce/woocommerce
# Product Collection: Remove automatic migration code

### Changes proposed in this Pull Request:

- Remove the code related to the automatic `Products (Beta)` -> `Product Collection` upgrade.
- Add an e2e test that ensures the manual upgrade works as expected.

Closes #42270

### How to test the changes in this Pull Request:

Ensure the manual upgrade works as expected:
1. Add the deprecated `Products (Beta)` block by pasting this snippet:

   ```html
   
   <div class="wp-block-query"></div>
   
   ```
2. Open the (Block) Settings panel and click the `Upgrade to Product Collection`.
3. Ensure the block properly upgraded to Product Collection.
